cpu/cortexm_common/include/irq_arch.h

```diff
@@ -60,6 +60,14 @@ static inline __attribute__((always_inline)) void irq_restore(
     __set_PRIMASK(state);
 }
 
+static inline __attribute__((always_inline)) int irq_is_enabled(void)
```

Comment:
Can you add a small comment so 
```suggestion
/**
 * @brief See if IRQs are currently enabled
 */
static inline __attribute__((always_inline)) int irq_is_enabled(void)
```